docs/aerodromes/Archerfield.md

```diff
@@ -121,8 +122,8 @@ Archerfield Tower will not alter an aircrafts tracking until the aircraft is wit
 #### Visual Tracking
 
 !!! example
-    **BN TCU** -> **AF ADC**: "Estimate, UJE via TVT time 02."  
-    **AF ADC** -> **BN TCU**: "UJE"
+    <span class="coldline">**BN TCU** -> **AF ADC**</span>: "Estimate, UJE via TVT time 02."  
```

Comment:
coord example needs to be changed to standard heads-up format (via TVT, UJE), as we try to get rid of the obsolete estimates, routings, etc

Reply:
Discussion for the team I think.  I've updated the metro Ds to follow this Estimate format for now, might be worth revisiting.